cloudevents/http/event.py

```diff
@@ -67,8 +67,11 @@ def __init__(self, attributes: typing.Dict[str, str], data: typing.Any = None):
                 f"Missing required keys: {required_set - self._attributes.keys()}"
             )
 
-    def __eq__(self, other):
-        return self.data == other.data and self._attributes == other._attributes
+    def __eq__(self, other: typing.Any) -> bool:
+        if isinstance(other, CloudEvent):
+            return self.data == other.data and self._attributes == other._attributes
+        else:
+            return False
```

Comment:
```suggestion
        if isinstance(other, CloudEvent):
            return self.data == other.data and self._attributes == other._attributes
        return False
```

I guess the `else` clause is excessive here.